x/supplier/types/params.go

```diff
@@ -50,30 +66,57 @@ func (p Params) Validate() error {
 		return err
 	}
 
+	if err := ValidateStakingFee(p.StakingFee); err != nil {
+		return err
+	}
+
 	return nil
 }
 
 // ValidateMinStake validates the MinStake param.
 func ValidateMinStake(minStakeAny any) error {
-	minStakeCoin, ok := minStakeAny.(*cosmostypes.Coin)
+	minStakeCoin, err := paramAsPositiveuPOKT(minStakeAny, "min_stake")
+	if err != nil {
+		return err
+	}
+
+	if minStakeCoin.IsZero() {
+		return ErrSupplierParamInvalid.Wrapf("min_stake amount must be greater than 0: got %s", minStakeCoin)
+	}
+
+	return nil
+}
+
+// ValidateStakingFee validates the StakingFee param.
+func ValidateStakingFee(stakingFeeAny any) error {
+	if _, err := paramAsPositiveuPOKT(stakingFeeAny, "staking_fee"); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// TODO_IN_THIS_COMMOT: godoc...
+func paramAsPositiveuPOKT(paramAny any, paramName string) (*cosmostypes.Coin, error) {
```

Comment:
```suggestion
func paramAsStrictlyPositiveuPOKT(paramAny any, paramName string) (*cosmostypes.Coin, error) {
```

Reply:
The distinction is not clear to me. Can you elaborate and/or share your rationale?

Reply:
My bad, I see it allows zero values.